resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -373,20 +377,17 @@ private[spark] class Client(
       .map(CompressionUtils.createTarGzip(_))
   }
 
-  private def getDriverLauncherService(
-      k8ClientConfig: Config,
-      kubernetesMaster: String): KubernetesSparkRestApi = {
+  private def getDriverLauncherService(k8ClientConfig: Config): KubernetesSparkRestApi = {
```

Comment:
If a user has an empty/invalid kube config file and specifies `k8s` as master, does the library handle that exception? I can't find that code path in this PR.